.github/workflows/automated-test-acceptances.yml

```diff
@@ -84,37 +84,7 @@ jobs:
           ACCTEST_TIMEOUT: ${{ vars.ACCTEST_TIMEOUT }}
           TEST_REGEX: "^TestAccConfig"
         run: make testacc
-
-  acceptance-tests-fed:
-    if: ${{ github.event.label.name == 'run-testacc' || github.event.label.name == 'run-testacc-project-fed' || github.ref == 'refs/heads/master' }}
-    needs: [ authorize ]
```

Comment:
`acceptance-tests-fed` is not running any tests (see [9473029508](https://github.com/mongodb/terraform-provider-mongodbatlas/actions/runs/5245497202/jobs/9473029508#step:4:130)) so I am deleting it

Reply:
yes, they all need AWS credentials and we do not provide them as part of the CI

Reply:
Can we create a JIRA ticket to add these at some point then?

Reply:
Sure. Created https://jira.mongodb.org/browse/INTMDB-887